docs/pages/model_hub/asr.md

```diff
@@ -49,4 +49,37 @@ Here are some other possible configurations for the Whisper deployment:
             compute_type=WhisperComputeType.FLOAT32,
         ).model_dump(mode="json"),
     )
-    ```
\ No newline at end of file
+    ```
+
+### Diarized ASR
+
+Diarized transcription can be generated by using [WhisperDeployment](./../../reference/deployments.md#aana.deployments.WhisperDeployment) and [PyannoteSpeakerDiarizationDeployment](./../../reference/deployments.md#aana.deployments.PyannoteSpeakerDiarizationDeployment) and combining the timelines using post processing with [PostProcessingForDiarizedAsr](./../../reference/processors.md#aana.processors.speaker.PostProcessingForDiarizedAsr).
+
+Example configuration for the PyannoteSpeakerDiarization model is available at [Speaker Diarization](./speaker_recognition.md/#speaker-diarization-sd-models) model hub.
+
+You can simply define the model deployments and the endpoint to transcribe the video with diarization. Below code snippet shows the how to combine the outputs from ASR and diarization deployments:
+
+
+```python
+from aana.processors.speaker import PostProcessingForDiarizedAsr
+
+# diarized transcript requires word_timestamps from ASR
+whisper_params.word_timestamps = True
+transcription = await self.asr_handle.transcribe(
+    audio=audio, params=whisper_params
+)
+diarized_output = await self.diar_handle.diarize(
+    audio=audio, params=diar_params
+)
+updated_segments = PostProcessingForDiarizedAsr(
+    diarized_segments=diarized_output["segments"],
+    transcription_segments=transcription["segments"],
+)
+output_segments = [
```

Comment:
Not sure if we need that filtering here. Instead I would add example output as a comment.